iterative_verifier.go

```diff
@@ -573,6 +614,29 @@ func (v *IterativeVerifier) compareFingerprints(pks []uint64, table *schema.Tabl
 		return nil, targetErr
 	}
 
+	mismatches := compareHashes(sourceHashes, targetHashes)
+	if len(mismatches) <= 0 {
+		return mismatches, nil
+	}
+
+	if v.CompressionVerifier != nil && v.CompressionVerifier.IsCompressedTable(table.Name) {
```

Comment:
Nitpick: Why not `if len(mismatches) > 0 && ...` here and then you can get rid of line 618?